src/filesystem/index.ts

```diff
@@ -565,6 +573,137 @@ server.setRequestHandler(CallToolRequestSchema, async (request) => {
         };
       }
 
+      case "load_folder": {
```

Comment:
Did you mean to leave this tool in? I don't see it in the README.

Reply:
Good catch thank you, I will remove it - I was using this personally but I think it's still a bit too janky to commit.

Many issues/questions:
Should it recurse?
Loading more than 5 files into context this way isn't supported by Claude so it frequently breaks